.travis.yml

```diff
@@ -20,10 +20,11 @@ env:
     # Variables found by default in Docker Hub builder
     - DOCKER_REPO=tecnativa/doodba
   matrix:
+    - DOCKER_TAG=7.0
     - DOCKER_TAG=8.0
-    - DOCKER_TAG=9.0
-    - DOCKER_TAG=10.0
-    - DOCKER_TAG=11.0
+    - DOCKER_TAG=9.0 PG_VERSIONS=9.6
+    - DOCKER_TAG=10.0 PG_VERSIONS=10
+    - DOCKER_TAG=11.0 PG_VERSIONS=11
     - DOCKER_TAG=12.0
```

Comment:
And why not here putting `PG_VERSIONS=11`?

Reply:
It's the default. I just leave room for future versions.